src/scaffoldmaker/meshtypes/meshtype_3d_wholebody2.py

```diff
@@ -0,0 +1,432 @@
+                    (10, [[5.250, -0.600, 0.000], [1.915, -0.427, 0.000], [0.039, 0.174, 0.000], [-0.148, -0.127, 0.004], [-0.000, 0.000, 0.210], [-0.002, -0.004, -0.055]]),
+                    (11, [[5.250, 0.600, 0.000], [1.915, 0.427, 0.000], [-0.039, 0.174, 0.000], [0.144, -0.115, -0.004], [0.000, -0.000, 0.210], [-0.002, 0.004, -0.055]]),
+                    (12, [[2.439, -1.399, 0.000], [0.920, -0.382, 0.000], [0.040, 0.096, 0.000], [-0.006, -0.008, 0.001], [-0.000, 0.000, 0.098], [0.004, -0.002, -0.021]]),
+                    (13, [[3.807, -1.755, 0.000], [1.181, -0.272, 0.078], [0.019, 0.084, -0.000], [-0.013, -0.044, 0.000], [-0.006, 0.001, 0.098], [-0.016, 0.003, -0.039]]),
+                    (14, [[4.841, -1.961, 0.137], [0.882, -0.141, 0.194], [0.002, 0.011, -0.000], [-0.016, -0.104, 0.001], [-0.004, 0.001, 0.021], [0.038, -0.008, -0.110]]),
+                    (15, [[2.439, 1.399, 0.000], [0.946, 0.385, 0.000], [-0.039, 0.096, 0.000], [0.008, -0.007, -0.001], [0.000, -0.000, 0.098], [0.004, 0.002, -0.022]]),
+                    (16, [[3.911, 1.761, 0.000], [1.211, 0.255, 0.091], [-0.018, 0.085, 0.000], [0.013, -0.039, -0.000], [-0.007, -0.002, 0.098], [-0.019, -0.004, -0.029]]),
+                    (17, [[4.932, 1.939, 0.153], [0.825, 0.101, 0.214], [-0.003, 0.020, 0.000], [0.010, -0.090, -0.002], [-0.010, -0.002, 0.041], [0.031, 0.006, -0.080]]),
+                    (18, [[8.100, -0.665, 0.000], [0.775, -0.027, 0.275], [0.005, 0.175, 0.002], [0.022, -0.000, 0.004], [-0.048, -0.000, 0.136], [-0.157, -0.004, -0.039]]),
+                    (19, [[8.459, -0.683, 0.298], [0.258, -0.010, 0.370], [0.003, 0.175, 0.003], [-0.005, 0.000, -0.009], [-0.127, 0.001, 0.088], [0.036, -0.004, -0.000]]),
+                    (20, [[8.601, -0.685, 0.694], [0.024, 0.006, 0.399], [-0.001, 0.175, -0.002], [-0.001, -0.000, 0.003], [-0.099, -0.000, 0.006], [0.077, 0.004, -0.084]]),
+                    (21, [[8.100, 0.665, 0.000], [0.775, 0.027, 0.275], [-0.005, 0.175, -0.002], [-0.022, -0.000, -0.006], [-0.048, 0.000, 0.136], [-0.157, 0.006, -0.039]]),
+                    (22, [[8.459, 0.683, 0.298], [0.258, 0.010, 0.370], [0.000, 0.175, -0.005], [0.005, 0.000, 0.009], [-0.127, 0.002, 0.089], [0.034, 0.004, 0.001]]),
+                    (23, [[8.601, 0.685, 0.694], [0.024, -0.006, 0.399], [0.001, 0.175, 0.002], [-0.005, 0.000, -0.003], [-0.104, 0.000, 0.006], [0.071, -0.010, -0.087]])
+                ]]),
+            "userAnnotationGroups": [
+                {
+                    '_AnnotationGroup': True,
+                    'dimension': 1,
+                    'identifierRanges': '1-3',
+                    'name': 'head',
+                    'ontId': 'None'
+                },
+                {
+                    '_AnnotationGroup': True,
+                    'dimension': 1,
+                    'identifierRanges': '6-8',
+                    'name': 'torso',
+                    'ontId': 'None'
+                },
```

Comment:
Use Uberon terms for head, torso. Is neck gone? See arm/leg below.
Also ensure terms are in annotation/body_terms.py

Reply:
Neck is lumped together with the head for now.